### PR TITLE
{data}[GCCcore/11.2.0] XML-Compile v1.63

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires:
* https://github.com/easybuilders/easybuild-easyconfigs/pull/15176

required by MEME (see #15169)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3531.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/5b6d1305edee72d5ac37c3e94f921a86 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node138.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/847942cb1d87e42f16934330234456fa for a full test report.

File: easybuild/easyconfigs/x/XML-Compile/XML-Compile-1.63-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    }),
+]
+
+modextrapaths = {
+    'PERL5LIB': 'lib/perl5/site_perl/%(perlver)s/',
+}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/perl5/site_perl/%(perlver)s/%(arch)s-linux-thread-multi/auto/XML/Compile'],
```
Comment (jfgrimm) [inline easybuild/easyconfigs/x/XML-Compile/XML-Compile-1.63-GCCcore-11.2.0.eb:58]: I'd probably check:
```suggestion
    'dirs': ['lib/perl5/site_perl/%(perlver)s/XML/Compile'],
```
which is where the `.pm` files are installed

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in total)
node138.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/bb6719b242e7f7e38f9d1e600d8fb53a for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node138.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/f09e6d8a48c3e4496b7234c910e44764 for a full test report.

**Review by jfgrimm — APPROVED** [review]
LGTM

**Pull request merged**